docs/GE_Requirements.md

```diff
@@ -148,23 +148,41 @@ contributions **MUST** include clauses as follows:
 
 ## Releases
 
--   Every release MUST have an associated Release Notes entry at the Github
-    Releases section. See
-    [Example](https://github.com/telefonicaid/fiware-orion/releases)
+A "FIWARE Release" shall contain versioned releases of all Generic Enablers
+found within the [catalogue](https://www.fiware.org/developers/catalogue/). All
+GEs pertaining to a FIWARE Release must work seamlessly with the corresponding
+FIWARE Context Broker part of such FIWARE Release.
 
-The following rules apply for numbering FIWARE releases:
+The following rules apply for numbering FIWARE Releases:
 
 -   Major release number: to be increased by decision at TSC level, typically
     linked with a major achievement (e.g., support to NGSI-LD).
 -   Minor release number: to be incremented with relevant milestones agreed at
-    TSC level. Milestones will be FIWARE Summits by default. Additional ones can
-    be added if strictly needed and agreed at TSC level.
--   Major.minor numbering of FIWARE GEs in the Core Context Management chapter
-    matches numbering of FIWARE release, i.e., they automatically increment as
-    FIWARE releases evolve.
+    TSC level. Milestones will be aligned with the celebration of FIWARE Summits
+    by default. Additional ones can be added if strictly needed and agreed at
+    TSC level.
+
+### GE Releases
 
-The release note for each versioned release should include a list of the latest
-version of interoperable enablers it has been tested against as follows:
+-   Everytime there is a FIWARE Release, GE Owners **MUST** tag the GE release
+    pertaining to such FIWARE Release with `FIWARE<major>.<minor>`. Please note
+    that this imply that certain GE releases could be double tagged. The
+    referred tag will be intended to mark clearly that such GE Release is part
+    of and compatible with the corresponding FIWARE Release. Docker images or
+    other associated artefacts **MUST** also be properly tagged with the
+    referred FIWARE tag.
+
+"Compatibility" with a release means compatibility with the FIWARE Context
```

Comment:
This implies that likely breaking changes within the Context Broker should be scheduled and announced beforehand (or more like roadmapped) to give GEs a chance to catch up.

For example removing the NGSI v1 interface could break compatibility so dev teams would need to be warned that they may have work to do.

Does this need to be added to the text somehow?

Reply:
added